WcaOnRails/app/views/teams/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<% provide(:title, 'Teams') %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <% can_edit = current_user.can_edit_users? %>
+  <% if can_edit %>
+    <%= link_to "New team", new_team_path %>
+  <% end %>
+
+  <% if @teams.empty? %>
+    <div class="alert alert-warning">There are currently no teams.</div>
+  <% else %>
+    <table class="table">
```

Comment:
nit: Could we use wca_table here instead to get consistent styling across the site?